src/routing/approved.rs

```diff
@@ -1046,45 +1044,12 @@ impl Approved {
                 message: MessageType::ClientMessage(ClientMessage::from(content)?),
             }]);
         }
-        if let SrcAuthority::BlsShare {
-            proof_share,
-            src_name,
-            ..
-        } = &src
-        {
-            let signed_bytes = bincode::serialize(&msg.signable_view())?;
-            match self
-                .message_accumulator
-                .add(&signed_bytes, proof_share.clone())
-            {
-                Ok(proof) => {
-                    trace!("Successfully aggregated signatures for message: {:?}", msg);
-                    let key = msg.proof_chain_last_key()?;
-                    if key.verify(&proof.signature, signed_bytes) {
-                        self.send_event(Event::MessageReceived {
-                            content,
-                            src: SrcLocation::Section(*src_name),
-                            dst,
-                        });
-                    } else {
-                        trace!(
-                            "Aggregated signature is invalid. Handling message {:?} skipped",
-                            msg
-                        );
-                    }
-                }
-                Err(AggregatorError::NotEnoughShares) => {}
-                Err(err) => {
-                    trace!("Error accumulating message at destination: {:?}", err);
-                }
-            }
-            return Ok(vec![]);
-        }
```

Comment:
Note for reviewers: Don't mind this removal. It should have been done in a previous PR but somehow slipped through. It's unrelated to the purpose of this PR.